packages/cli/src/scripts/status.js

```diff
@@ -81,44 +80,46 @@ async function contractsInfo(controller) {
     .forEach(contractAlias => log.warn(`- ${contractAlias} will be removed on next push`));
 }
 
-async function stdlibInfo(networkFile) {
+async function depsInfo(networkFile) {
```

Comment:
As we are using `dependencies`/`dependency` everywhere, we should rename this to `dependenciesInfo` for consistency.

Reply:
@jcarpanelli updated!